src/css/base.scss

```diff
@@ -1,7 +1,181 @@
 /* This is an example of using Sass, otherwise plain CSS works too*/
-$primary-background: radial-gradient(circle, rgba(63,94,251,1) 0%, rgba(252,70,107,1) 100%);
-;
+
+/* Color Theme Swatches in HSLA */
```

Comment:
Need to consolidate these color options (and possible rethink the variable names) when setting up partials. I am going to leave them as they are for now.